201-traffic-manager-webapp/azuredeploy.json

```diff
@@ -1,99 +1,93 @@
 {
-    "$schema": "https://schema.management.azure.com/schemas/2015-01-01/deploymentTemplate.json",
-    "contentVersion": "1.0.0.0",
-    "parameters": {
-        "uniqueDnsName": {
-            "type": "string",
-            "metadata": {
-                "description": "Relative DNS name for the traffic manager profile, resulting FQDN will be <uniqueDnsName>.trafficmanager.net, must be globally unique."
-            }
-        },
-        "uniqueDnsNameForWebApp": {
-            "type": "string",
-            "metadata": {
-                "description": "Relative DNS name for the WebApps, must be globally unique.  An index will be appended for each Web App."
-            }
-        }
+  "$schema": "https://schema.management.azure.com/schemas/2015-01-01/deploymentTemplate.json",
+  "contentVersion": "1.0.0.0",
+  "parameters": {
+    "uniqueDnsName": {
+      "type": "string",
+      "metadata": {
+        "description": "Relative DNS name for the traffic manager profile, resulting FQDN will be <uniqueDnsName>.trafficmanager.net, must be globally unique."
+      }
     },
+    "uniqueDnsNameForWebApp": {
```

Comment:
since TM is the endpoint, do we need to have the user supply a name?  Or can we just generate one?